src/components/RenderError.js

```diff
@@ -14,6 +14,9 @@ import ExportLogsBtn from 'components/ExportLogsBtn'
 import Box from 'components/base/Box'
 import Space from 'components/base/Space'
 import Button from 'components/base/Button'
+import IconTriangleWarning from 'icons/TriangleWarning'
+import ConfirmModal from './base/Modal/ConfirmModal'
```

Comment:
better to keep consistency with `components/base/Modal/ConfirmModal` ..

Reply:
lets merge and we'll change them all later

Reply:
not again 😁